cirq-google/cirq_google/circuit_serializer_test.py

```diff
@@ -0,0 +1,673 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict
+import pytest
+import sympy
+from google.protobuf import json_format
+
+import cirq
+import cirq_google as cg
+from cirq_google.api import v2
+
+
+def op_proto(json: Dict) -> v2.program_pb2.Operation:
+    op = v2.program_pb2.Operation()
+    json_format.ParseDict(json, op)
+    return op
+
+
+Q0 = cirq.GridQubit(2, 4)
+Q1 = cirq.GridQubit(2, 5)
+
+
+X_PROTO = op_proto(
+    {
+        'xpowgate': {'exponent': {'float_value': 1.0}},
+        'qubit_constant_index': [0],
+    }
+)
+
+
+OPERATIONS = [
```

Comment:
Would it be possible to include symbol cases here and in the CircuitOperations tests ?

Reply:
Done.  CircuitOperations with symbols still pending.

Reply:
Modified circuit operation test to include symbols.